apps/zotonic_launcher/src/command/zotonic_cmd_addsite.erl

```diff
@@ -193,14 +196,19 @@ parse_args([ "-n", Schema | Args ], Acc) ->
 parse_args([ "-a", Pw | Args ], Acc) ->
     parse_args(Args, Acc#{ admin_password => Pw });
 parse_args([ "-A", App | Args ], Acc) ->
-    case re:run(App, "^(true|false|1|0)$", [caseless, {capture, none}]) of
-        match ->
-            Bool = z_convert:to_bool(string:to_lower(App)),
-            parse_args(Args, Acc#{ app => Bool });
-        nomatch ->
-            {error, App}
-    end;
+    parse_bool_arg(app, App, Args, Acc);
+parse_args([ "-U", Umbrella | Args ], Acc) ->
+    parse_bool_arg(umbrella, Umbrella, Args, Acc);
 parse_args([ "-" ++ _ = Arg | _ ], _Acc) ->
     {error, Arg};
 parse_args(Rest, Acc) ->
     {ok, {Acc, Rest}}.
+
+parse_bool_arg(Key, Arg, Args, Acc) ->
+    case re:run(Arg, "^(true|false|1|0)$", [caseless, {capture, none}]) of
```

Comment:
```erlang
re:run("ц", "^(true|false|1|0)$", [caseless, {capture, none}]). ** exception error: bad argument
     in function  re:run/3
        called as re:run([1094],
                         "^(true|false|1|0)$",
                         [caseless,{capture,none}])
        *** argument 1: not an iodata term
```
Here will be a crash with try to parse `unicode` string. Maybe it make sense to add `catch` here.